homeassistant/components/vicare/strings.json

```diff
@@ -65,6 +65,19 @@
         "name": "Heating"
       }
     },
+    "fan": {
+      "ventilation": {
+        "name": "Ventilation",
+        "state_attributes": {
```

Comment:
Translations for the extra state attributes are missing

Reply:
I don't see the extra state attributes, have they been removed?

Reply:
Yes, they were removed.